fog/overseer/server/src/worker.rs

```diff
@@ -472,8 +497,8 @@ where
                             let number_of_remaining_tries =
                                 Self::NUMBER_OF_TRIES - current_try as usize;
                             let error_message = match number_of_remaining_tries {
-                                0 => format!("Did not succeed in setting a new key on node at index {}. Underlying error: {}", i, err),
-                                _ => format!("New keys were not successfully set on the ingest node at index {}. Will try {} more times. Underlying error: {}", i, number_of_remaining_tries, err),
+                                0 => format!("Did not succeed in setting a new key on node {}. Underlying error: {}", ingest_client.get_uri(), err),
+                                _ => format!("New keys were not successfully set on the ingest node {}. Will try {} more times. Underlying error: {}", ingest_client.get_uri(), number_of_remaining_tries, err),
```

Comment:
Nit: This variant seems strictly more informative. How about we replace this `match` with the one `format!` call?

Reply:
Done.